docs/tasks/administer-cluster/cpu-management-policies.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Control CPU Management Policies on the Node
+---
+
+* TOC
+{:toc}
+
+Kubernetes keeps many aspects of how the pod executes on the node abstracted
+away from the user. This is by design.  However, some workloads require
+stronger guarantees in terms of latency and/or performance in order to operate
+acceptably.  The kubelet provides methods for enabling more complex workload
+placement policies while keeping the abstraction free from explicit placement
+directives.
+
+## CPU Management Policies
+
+By default, the kubelet uses [CFS quota](https://en.wikipedia.org/wiki/Completely_Fair_Scheduler)
+to enforce pod CPU limits.  When the node is running many CPU bound pods, this
+can result in the workload being moved to different CPU cores depending on
+whether the pod is throttled or not and which CPU cores are available at
+scheduling time.  Many workloads are not sensitive to this migration and thus
+work fine without any intervention.
+
+However, in workloads where CPU cache affinity and scheduling latency
+significantly affects workload performance, the kubelet allows alternative CPU
```

Comment:
...the kubelet allows alternative CPU management policies to determine some placement preferences on the node.